doc/tools/make_rst.py

```diff
@@ -842,6 +872,15 @@ def make_rst_class(class_def: ClassDef, state: State, dry_run: bool, output_dir:
             + "\n\n"
         )
 
+    if class_def.name in CLASSES_WITH_CSHARP_DIFFERENCES:
+        f.write(".. warning::\n\n\t")
```

Comment:
Maybe a `warning` is too strong? Maybe it should be `note` or `tip`?

Reply:
`note` sounds good.